pyproject.toml

```diff
@@ -0,0 +1,172 @@
+[build-system]
+requires = ["setuptools>=62.6", "versioneer[toml]==0.28"]
```

Comment:
Yes, what's your plan with that?

Reply:
Great, how does one have them programmatically installed? I ask because when I went to build a distribution with `python setup.py sdist bdist_wheel` (pretending I was going to release this branch) I got an error about not having `versioneer` installed. It was an easy fix -- I just manually installed `versioneer=0.28`. But if there was a way to make sure I always had the appropriate build requirements, that would be nice

Reply:
ah you should call `pipx run build` it will create an sdist and a wheel for you

Reply:
Yeah generally there is a move away from `setup.py` altogether. So using one of the other tools and not calling `setup.py` is the way to go

Reply:
I updated the release procedure with these instructions https://github.com/dask/dask/pull/10102/files#diff-e3320cf793111444adc8e050d2312bed4907b50d71006ff68c7207f041ef978fR45